packages/yourloops/lib/team/hook.tsx

```diff
@@ -30,7 +30,7 @@ import React from "react";
 import _ from "lodash";
```

Comment:
I think we can remove lodash from this file

Reply:
Out of scope. Still being used in code that is not related with what I'm doing here